### PR TITLE
Added variations on three books

### PR DESCRIPTION
**Pull request by Epictyphlosion**
#23165 
I added variations to the war novel, coming of age novel, and crime novel. These variations have different descriptions and slightly different values

### CODE REVIEW

File: data/json/items/book/misc.json [hunk 1]
```diff
@@ -562,7 +562,24 @@
     "chapters": 16,
     "fun": 2
   },
-  {
+  {
```
Comment (ZhilkinSerg) [inline data/json/items/book/misc.json]: Some strange character used to break lines here and below.

File: data/json/items/book/misc.json [hunk 2]
```diff
@@ -578,6 +595,22 @@
     "chapters": 24,
     "fun": 3
   },
+  {
+    "id": "novel_crime2",
+    "type": "BOOK",
+    "name": "crime_novel",
+    "description": "A story about three people in the fictional city of Los Santos, a gangster, an upper-class man, and a psychopath, eventually banding together to pull-off the biggest heist in the city's history"
```
Comment (Cyrano7) [inline data/json/items/book/misc.json]: You need a comma at the end of this line.

**Comment by illi-kun** [comment]
@Epictyphlosion it's great to see you as a first-time contributor, I hope your patience will help you to move on dev side :-)

I guess you've created this pr via web interface but that's not the best solution because
1) PRs have to be tested on your side before publishing it here. Testing it can help you to catch errors like one pointed by Cyrano7 above.
2) Your browser (or your text editor if I was wrong in my guess) add some weird invisible characters at the end of lines you've touched. Take a look:

whitespaces

These characters are also highlighted by github, btw.

So the proper way is to install some git soft, fork a repo and get its copy on your local machine. https://help.github.com/ is very useful for this part of work.

For working on code you need a compiler but if you plan to publish only json changes then it's ok to take a pre-compiled application from latest experimental version and run it for checking your changes.

Anyway, you need a good text editor, notepad++ is a really good one for windows and atom is my choice for linux.

Let me know about your progress, and if you are ready to keep doing all this dev stuff, we will continue.

**Comment by Epictyphlosion** [comment]
I might add other books, and items as well. This is not just to add some variety, but also because most of the files in the book folder haven't been touched in 2 whole years!

**Comment by John-Candlebury** [comment]
Its possible for the same item to support several randomly chosen description. This would save you the chore of copying and pasting tens of definitions.

You can see the Jain definition for "tshirt_text" for an example of how it works

**Comment by Epictyphlosion** [comment]
@John-Candlebury It's not just descriptions that I'm changing, it's the values of them as well. I might change the titles to actual book titles as well

**Comment by John-Candlebury** [comment]
Also make sure to add them to an item group, right now, your books don't spawn anywhere

**Comment by Epictyphlosion** [comment]
Do I have to edit the item group file, or just this one?

**Comment by Epictyphlosion** [comment]
Made another commit so Travis can check it again

**Comment by John-Candlebury** [comment]
Look in item_groups.json. If you search magazines you'll find an entry with an enormous list of books.. Add you item ids to that list and they will spawn in most places with recreational books.

**Pull request merged**